seebeyondtests/build.xml

```diff
@@ -1,25 +1,41 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-
 <!DOCTYPE project [
```

Comment:
I don't think the comment can come in the middle of the DOCTYPE.  Try combining the DOCTYPE into one line, even though it won't look as nice.  That way the copyright plugin won't mess with it.